cachi2/core/package_managers/bundler/main.py

```diff
@@ -0,0 +1,16 @@
+from cachi2.core.models.input import Request
+from cachi2.core.models.output import EnvironmentVariable, ProjectFile, RequestOutput
+from cachi2.core.models.sbom import Component
+
+
+def fetch_bundler_source(request: Request) -> RequestOutput:
+    """Resolve and process all bundler packages."""
+    components: list[Component] = []
```

Comment:
Do you plan to eventually use these lists (L8:L10) in this function?

Reply:
yes

Reply:
But they will be populated with some data :smile:  I should mention

Reply:
Thank you, they do look a bit suspicious, so I had to ask.